### PR TITLE
Fix for Spotify 1.1.67

### LINKED ISSUE #1029: Spicetify Custom App react symbol error
## ℹ Computer information

- Spotify version: Spotify for Windows 1.1.67.586.gbb5ef64e
- Spicetify version: 2.6.1

## 📝 Provide detailed reproduction steps (if any)

1. Try to apply spicetify after upgrading to this Spotify version
2. errors with custom app react symbols

### ✔️ Expected result

_What is the expected result of the above steps?_

Spicetify to apply

### ❌ Actual result

_What is the actual result of the above steps?_

Errors applying spicetify. To note, Spotify doesnt restart and Im still able to use it, but wont apply the config

## 📷 Screenshots

_Are there any useful screenshots?_
imagen

Spicetify config state:
imagen

### PR DESCRIPTION
**Pull request by itsmeow**
Fixes #1029 #1020 #980 #983 #989 #1003 #1039 #1022

### Fixes

- Error: Custom React App Symbols not found
- `Spicetify.React`
- `Spicetify.Platform`
- `Spicetify.ReactComponent.RightClickMenu`
- `Spicetify.ReactComponent.ContextMenu`
- `Spicetify.ReactComponent.Menu`
- Lyrics Plus (changes in internal APIs)

### Known issues/broken hooks

- `Spicetify.Menu`
- CSS mapping issues

On another note, the list of broken hooks from #945 under `Spicetify.ReactComponent` has fixed itself, so that's one less thing to worry about.

### CODE REVIEW

**Comment by JulienMaille** [comment]
Excellent! Is there a way I can test it on my themes before it's merged?

**Comment by RobzHD** [comment]
Edited the lines as shown in both files, but still getting the same result. Is there something else I have to do?

**Comment by th3an7** [comment]
> Excellent! Is there a way I can test it on my themes before it's merged? I mean without compiling go myself

Hi,
Sorry for hijacking and a little bit of offtopic...

I've set up Github Action workflow that automatically builds new `spicetify.exe` which should be enough to test changes without compiling it on your own - just download artifact and replace `spicetify.exe` with auto-builded one

https://nightly.link/th3an7/spicetify-cli/workflows/main/67-fixes/spicetify.zip

Whole workflow `.yml` file available [here](https://github.com/th3an7/spicetify-cli/blob/67-fixes/.github/workflows/main.yml)

**Comment by danielphan2003** [comment]
I tested this build out with the following config:
- Extensions: ddt, ddt-dynamic, Vibrant.min.js, fullAppDisplay, loopyLoop, popupLyrics, shuffle+, trashbin.
  ddt, ddt-dynamic, Vibrant.min.js is the [Dribbblish Dynamic Theme][ddt-theme].
- Custom apps: lyrics-plus, new-releases, reddit.
- Theme: ddt (colorscheme is `base`).

After reapplying, Spotify shows this error: 

[Here][build-log] is the build log.

[ddt-theme]: https://github.com/JulienMaille/spicetify-dynamic-theme
[build-log]: https://catbin.up.railway.app/momuyphupe

**Comment by itsmeow** [comment]
@danielphan2003 The Platform hook is still broken which is why this is a Draft PR, my Spotify doesn't work either with minimal extensions.

**Comment by itsmeow** [comment]
@RobzHD You're editing the source, you have to actually build and replace your spicetify. It doesn't fully work anyway, so just wait.

**Comment by itsmeow** [comment]
Good news!!!
image

This is on 1.1.67!!!

File: CustomApps/lyrics-plus/Utils.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class Utils {
     static addQueueListener(callback) {
-        Spicetify.Player.origin2.state.addExtendedStatusListener(callback);
+        Spicetify.Player.origin2.state.addQueueListener(callback);
     }
 
     static removeQueueListener(callback) {
-        Spicetify.Player.origin2.state.removeExtendedStatusListener(callback);
+        Spicetify.Player.origin2.state.queueListeners =
+          Spicetify.Player.origin2.state.queueListeners.filter(v => v == callback);
```
Comment (daksh2k) [inline CustomApps/lyrics-plus/Utils.js:8]: I think this should be 
`Spicetify.Player.origin2.state.queueListeners = Spicetify.Player.origin2.state.queueListeners.filter(v => v != callback);`
As we are removing that listener.

Comment (itsmeow) [reply CustomApps/lyrics-plus/Utils.js:8]: Correct! Good catch!

**Comment by Alatrix** [comment]
> 
> 
> Good news!!!
> This is on 1.1.67!!!

So how can we use it in spotify 1.1.67? Just wait?

**Comment by itsmeow** [comment]
@afonsojramos Will merge the PR, he is just unavailable right now and unable to review the PR.

**Review by afonsojramos — APPROVED** [review]

**Comment by SebiTalent04** [comment]
so this fix will be available in 2.6.3?

**Comment by TheGeeKing** [comment]
Normally

**Comment by Alatrix** [comment]
Now do we have to wait until next version is released or any other option?

**Comment by brenmadeit** [comment]
any stable versions I can roll back?

**Comment by TheGeeKing** [comment]
The last stable versions are Spotify 1.1.65.643 and Spicetify 2.6

**Comment by brenmadeit** [comment]
> The last stable versions are Spotify 1.1.65.643 and Spicetify 2.6

but some extension doesnt work for me and the dribbblish theme looks deformed

**Pull request merged**